ecosystem/lava-sdk/src/stateTracker/stateQuery/state_chain_query.ts

```diff
@@ -142,15 +183,33 @@ export class StateChainQuery {
         throw StateTrackerErrors.errTimeTillNextEpochMissing;
       }
 
+      this.latestEpoch = currentEpoch;
+
+      // in case of emergency mode, timeLeftToNextPairing is equal to epochDuration from downtimeParams
+      if (virtualEpoch != 0 && downtimeParams != undefined) {
+        const epochDuration = downtimeParams.getEpochDuration();
+        if (epochDuration != undefined) {
+          timeLeftToNextPairing = epochDuration.getSeconds();
```

Comment:
should be before the check:
if (timeLeftToNextPairing == undefined) {
        throw StateTrackerErrors.errTimeTillNextEpochMissing;
      }